### ISSUE TITLE
midi mapping editor cc changes don't register

### ISSUE THREAD

**Issue by williamthazard** [ryleelyman/seamstress#124]
if I've mapped a parameter to a midi cc and then decide I'd like to change which cc the parameter is mapped to, I go to the pmap editor and alter the cc number. When I hit delete to exit the pmap editor, the correct cc (i.e., the one I've changed it to in the editor) is displayed next to the parameter. But when I turn the knob to change the cc, the parameter doesn't change. Instead, the previous (incorrectly mapped) knob still changes the parameter.

tl;dr I'm pretty sure that while midi ccs _appear_ to change when they're edited in the pmap editor, the mapping does not _actually_ change
